0.data-download/README.md

```diff
@@ -9,20 +9,22 @@ https://depmap.org/portal/download/all/
 
 ## Files used in our multivariate gene dependency project
 
-### CRISPR_gene_dependency.csv
+### CRISPRGeneDependency.csv
 
 This data comprises scores from individual CRISPR gene knockout screens in cancer cell lines for every gene across many different ages and cancer types.
 The values represent gene dependency probability estimates for cell survival and growth for all models in the integrated gene effect.
 The higher the score, the greater the gene dependency. 
 
 The probability estimate is derived from the CRISPRGeneEffect estimates for all models which were introduced to an algorithm, Chronos, to infer gene knockout fitness effects based on an explicit model of cell proliferation dynamics after CRISPR gene knockout.
 
-Columns: gene
-Rows: the ModelID
+Columns (18,333): gene
 
-### sample_info.csv
+Rows (1,087): the ModelID
 
-This file gives detailed information on the cell lines, type of cancer, sex, and age of the patient.
+### Model.csv
```

Comment:
Updated file name based on new data set (22Q4)

Reply:
Did the row and column counts change?

Reply:
Ah, yes they did. I need to update the row and column numbers from both files, thanks!